### DISCUSSION TITLE
Add custom px Settings

### DISCUSSION THREAD

**Discussion by lx815** [tailwindlabs/tailwindcss#5167] [Ideas]
In actual development, it is often the case that the developer needs to set a fixed PX value. TailWindcss currently does not support this, so we have to write our own styles. Hopefully, I can add this to it, so if I want to define 100PX myself, I'll just write w-100px.

**Comment by aymeric35** [comment]
I think you can already do that, see: https://tailwindcss.com/docs/just-in-time-mode#arbitrary-value-support
So if I take your example with `w-100px`, it would be `w-[100px]`

**Comment by simonswiss** [comment]
Hey!

You can either extend your `width` object in the Tailwind config file like so...

```diff
module.exports = {
  theme: {
    extend: {
+      width: {
+       '100px': '100px',
+      },
    },
  },
}
```

... which will give you a `w-100px` class.

Or, as @aymeric35 mentioned, if you're using the `JIT` mode, you can pass an arbitrary value directly in your HTML like so:

```html
<div class="... w-[200px]">
```

...which will generate a 200px wide element.

Here's a Tailwind Play with both demos:

https://play.tailwindcss.com/Dj0epDQdqZ

![image](https://user-images.githubusercontent.com/485747/128673157-34deeb54-bea0-42c5-bcf4-e1617fa3dc9a.png)

Hope it helps!
